src/taxonomy/api/hooks/api.test.js

```diff
@@ -1,15 +1,24 @@
 import { useQuery } from '@tanstack/react-query';
-import useTaxonomyListData from './api';
+import { useTaxonomyListData, exportTaxonomy } from './api';
+
+const mockHttpClient = {
+  get: jest.fn(),
+};
 
 jest.mock('@tanstack/react-query', () => ({
   useQuery: jest.fn(),
+  useMutation: jest.fn(),
```

Comment:
I think we don't need this

Reply:
Updated here 0be804c079df1e8964355b265cee6f02afb3335d